libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts

```diff
@@ -390,36 +408,45 @@ export class SdsFiltersComponent implements OnInit, OnChanges {
       }
 
       field.hide = false;
-    })
+    });
   }
 
   /**
    * Create chips to display for horizontal filters based on current model
    * and formly field config
-   * @param model 
-   * @param fields 
+   * @param model
+   * @param fields
    */
   private generateChips(model: any, fields: FormlyFieldConfig[]) {
-    const readonlyData = FormlyUtilsService.formlyConfigToReadonlyData(fields, model);
-    const chipsWithValue = readonlyData.filter(data => data.value);
+    const readonlyData = FormlyUtilsService.formlyConfigToReadonlyData(
+      fields,
+      model
+    );
+    const chipsWithValue = readonlyData.filter((data) => data.value);
     let allChips = [];
-    chipsWithValue.forEach(chip => {
+    chipsWithValue.forEach((chip) => {
       if (typeof chip.value != 'object') {
+        chip['srValue'] = chip.value;
```

Comment:
@yerramshilpa can you confirm for me that this is the only code change that was made for this issue? The rest appear to be formatting changes, but I want to make sure I'm not missing another code change.

Reply:
Yes just one line, rest is formatting